apps/judicial-system/backend/src/app/modules/notification/test/notificationController/sendCaseFilesUpdatedNotifications.spec.ts

```
@@ -0,0 +1,69 @@
+    const mockSendMessagesToQueue =
+      messageService.sendMessagesToQueue as jest.Mock
+    mockSendMessagesToQueue.mockResolvedValue(undefined)
+
+    givenWhenThen = async (caseId) => {
+      const then = {} as Then
+
+      await notificationController
+        .sendCaseNotification(caseId, user, { id: caseId } as Case, {
+          type: NotificationType.CASE_FILES_UPDATED,
+        })
+        .then((result) => (then.result = result))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)